app/locales/en/translations.js

```diff
@@ -248,6 +248,11 @@ export default {
             support: `Support`,
             contact: `Contact`
         },
+        'permission-language':{
```

Comment:
We don't know what the other providers are going to want as far as trademark info. They're not all going to be arxiv based, but it is likely they're going to want some sort of IP statement if they are branding somehow. We can roll it into some more arbitrary text location if Product approves that, but it's otherwise having this be flexible is better.